src/App.js

```diff
@@ -2,19 +2,28 @@ import React, { Component } from 'react';
 import './App.css';
 import InputSubmission from './InputSubmission';
 import ZipsList from './ZipsList';
+// import FetchResults from './FetchResults';
+// import axios from 'axios';
 
 class App extends Component {
   constructor(){
     super();
     
     this.state = { zips: [] };
+    // this.state = { zips: [], results: [] };
   
 }
 componentDidMount() {
   let zipsFromStorage = localStorage.getItem('zips');
   zipsFromStorage = JSON.parse(zipsFromStorage);
 
-  this.setState({zips: zipsFromStorage ? zipsFromStorage : []});
+  this.setState({zips: zipsFromStorage ? zipsFromStorage : [], results: []});
```

Comment:
Thanks Matt! It's gotten more complicated now, but that was helpful feedback. I'll refactor so it can look like this again. :)